### ISSUE TITLE
"Couldn't find an open pull request for branch with head at ."

### ISSUE THREAD

**Issue by hughgrigg** [unsplash/comment-on-pr#8]
Getting this error from `unsplash/comment-on-pr@master`:

```
Couldn't find an open pull request for branch with head at .
##[error]Docker run failed with exit code 1
```

The step trying to use `comment-on-pr` looka like this:

```
- uses: actions/checkout@v1
- name: Comment on zero test changes
        uses: unsplash/comment-on-pr@master
        if: steps.file_counts.outputs.test_file_count == 0
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          msg: >
            @${{ github.actor }}
            Please add tests.
```

**Comment by hughgrigg** [comment]
Seems this was due to some issue in `actions/checkout@v1`. Using `actions/checkout@master` seems to have fixed it.

**Comment by hughgrigg** [comment]
I think this might just happen when a new push is made to the PR branch before this step tries to make the comment.

**Comment by hughgrigg** [comment]
Actually same issue as https://github.com/unsplash/comment-on-pr/issues/7

**Comment by FarhanSajid1** [comment]
So what is the solution to this?

**Comment by aaronklaassen** [comment]
Fixed by #10
